containers/azure-terraform/README.md

```diff
@@ -50,6 +57,45 @@ Beyond `git`, this `Dockerfile` includes `zsh`, [Oh My Zsh!](https://ohmyz.sh/),
 
 5. Finally, press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** to start using the definition.
 
+## Testing the Definition
+
+This definition includes some test code that will help you verify it is working as expected on your system. Follow these steps:
+
+1. If this is your first time using a development container, follow [these steps](https://code.visualstudio.com/docs/remote/containers#_getting-started) to setup your machine.
```

Comment:
This feels like a duplicate for #1 in the "Adding the definition to your project".  It also includes a different link (direct link vs. aka.ms). 

Would it be possible to make this section build on the steps above instead of starting over?

Reply:
Agreed, updated to make it simpler and less repetitive.